main.js

```diff
@@ -0,0 +1,25 @@
+const table = document.getElementById('table');
+
+function sortTr(event) {
+    const { rows } = table.tBodies[0];
+    const { cellIndex } = event.target;
+    const { type } = event.target.dataset;
+    const body = table.tBodies[0];
+    let newRowsTable = [];
+
+    for (let row of rows) {
+        newRowsTable = [...newRowsTable, row];
+    }
+
+    if (type === 'number') {
+        newRowsTable.sort((rowFirst, rowSecond) => (+rowFirst.cells[cellIndex].textContent)
+            - (+rowSecond.cells[cellIndex].textContent));

```

Comment:
bad formatting
it is hard to read. Will be better to save `.cells[cellIndex].textContent` in variables and use them inside both ifs.
And the line will be not too long